packages/polymarket-notifier/src/abi/abi.json

```diff
@@ -0,0 +1,830 @@
+[
```

Comment:
can we get this from the poly market package? do they export their contract artifacts?

Reply:
if not then we should only define the exact interface you need....this is super bloated otherwise.